Meadowlark-js/tests/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@edfi/meadowlark-tests",
+  "main": "dist/index.js",
+  "version": "0.2.0-dev.0",
+  "description": "Meadowlark system and integration tests",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
Put this at the same level as packages as it's own package but under "tests". This lets provides a level of detachment so they import core and backends rather that reach into those codebases. Marked as `private: true` to prevent accidental publication to the Ed-Fi npm registry. As a backup, defines the Ed-Fi registry for publish anyway, to avoid accidental publication to the main npm registry.